messages/bulkIngest.md

```diff
@@ -0,0 +1,35 @@
+# export.resume
+
+Run "sf %s --job-id %s" to resume the operation.
```

Comment:
can't use the `<%= config.bin %>` placeholder here because this text isn't resolved by oclif like in commands.